libs/damap/src/lib/components/plans/plans.component.ts

```diff
@@ -50,31 +59,51 @@ export class PlansComponent implements OnInit {
   }
 
   getDocument(id: number) {
-    this.dialog.open(ExportWarningDialogComponent).afterClosed().subscribe(
-      _ => this.backendService.getDmpDocument(id)
-    );
+    this.backendService.getDmpById(id).subscribe(x => {
+      this.openExportWarningDialog(x.project?.funderSupported, id);
+    });
+  }
+
+  openExportWarningDialog(funderSupported: boolean, id: number): void {
+    const dialogRef = this.dialog.open(ExportWarningDialogComponent, {});
+    dialogRef.componentInstance.funderSupported = funderSupported;
+
+    dialogRef.beforeClosed().subscribe(result => {
+      if (result === undefined) {
+        return;
+      } else {
```

Comment:
Else can be removed, as we return in the if before. Also reduces complexity when reading code